subworkflows/local/quant.nf

```diff
@@ -0,0 +1,78 @@
+include { OPENMS_IDRIPPER }                                                 from '../../modules/local/openms_idripper'
```

Comment:
Add a few lines of comment on what the sub-workflow does, see pre-existing sub-workflows